PRIVACY_POLICY.md

```diff
@@ -0,0 +1,65 @@
+# Swarm-Extension Privacy Policy
+
+The Swarm Extension belongs to the Swarm Foundation with the purpose of offering web3 environment
+for running decentralised applications (dApps). It means, it provides similar functionalities as web2 browsers,
+so sensitive information may be handled by the browser-extension such as Bee client configuration
+or cross-domain localStore. Although the extension handles these sensitive information for
+the basic security and functionality of dApps, the Swarm Foundation does not share any of these data
+with other 3rd parties, nor collect it.
+
+By installing Swarm Extension, you accept this privacy policy.
+
+## Do we collect any information from you?
+
+We do not collect any data from you, including personal data.
+
+## Sharing Data By The Help of The Extension
+
+In Swarm Extension, you can set the connection configuration of an Ethereum Swarm
+Bee client node (that is a P2P storage client). The dApps can utilize this by calling
+the interfaces of the extension and perform read and write operations on the P2P storage network.
+The information stored on this network is publicly available and the encryption handling of
+such content is the dApps responsibility.
+
+## Under The Hood
+
+DApp developers can store user session data to the user's swarm-extension storage.
+It provides a cross-domain localstorage functionality that dApps can utilize
+in order to save and load the same session data on any gateway host.
+If dApps use the extension's private localstorage, it depends on the dapp in question
+what data will be stored and how, like also in the case of data storing on Swarm network.
+The extension does not share the userdata with other 3rd party, or collect it.
```

Comment:
Can dApps potentially access each other's data, that is stored in the extension? And can the user control that?

Reply:
depends on what kind of data.
dApp cannot access each other's localStorage, but they upload unencrypted data to Swarm (by the user), then other dApps can read it.